enzyme/CMakeLists.txt

```diff
@@ -128,6 +128,11 @@ if (ENZYME_FLANG)
   add_definitions(-DFLANG=1)
 endif()
 
+option(ENZYME_ROCM "Build for non-version compliant ROCM" OFF)
```

Comment:
grrml

Reply:
Can we document which version this is? Is this ROCM 4.X or 5?

Reply:
5.1.1